installers/helm/postgres-operator/templates/values_configmap.yaml

```diff
@@ -15,6 +15,8 @@ data:
     backrest_aws_s3_key: "{{ .Values.backrest_aws_s3_key}}"
     backrest_aws_s3_region: "{{ .Values.backrest_aws_s3_region}}"
     backrest_aws_s3_secret: "{{ .Values.backrest_aws_s3_secret}}"
+    backrest_aws_s3_uri_style: "{{ .Values.backrest_aws_s3_uri_style }}"
+    backrest_aws_s3_verify_tls: "{{ .Values.backrest_aws_s3_verify_tls }}"
```

Comment:
This is a YAML string, but in the other installers, it is a YAML boolean. 🤔 ​ I'm not sure there's any change necessary but thought I should point it out.

Reply:
Good point - I left it in quotes since it was a templated value.. Not sure if those are necessary or not.

Reply:
They are not necessary, as the template is being transformed into a YAML file.

Reply:
alright i remove double quotes from around the bool in these two templates. I used helm to template the values and the yaml no longer has double quotes